src/zcl_abapgit_objects.clas.abap

```diff
@@ -784,6 +793,16 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
 
     zcl_abapgit_objects_activation=>activate( is_step-is_ddic ).
 
+    IF lt_postprocess_items[] IS NOT INITIAL.
+
+*     Call postprocessing
+      li_exit = zcl_abapgit_exit=>get_instance( ).
+
+      li_exit->deserialize_postprocess( it_items = lt_postprocess_items[]
```

Comment:
just pass `is_step` to the exit? so there is no assumptions regarding abapGit implementation and future use

Reply:
Reason why I have introduced lt_postprocess_items is to capture the objects which were deserialized successfully.
May be I just remove this CHECK is_step-is_ddic = abap_false.??

Reply:
successes and errors are recorded to the log, so the exit already knows the objects which were deserialized successfully

```
          ii_log->add_success( iv_msg = |Object { <ls_obj>-item-obj_name } imported|
                               is_item = <ls_obj>-item ).
```

Reply:
IS_STEP is passed for the postprocessing userexit.